### PR TITLE
data: Jayanth Kumar

### PR DESCRIPTION
**Pull request by SJ-Kumar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4666"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @SJ-Kumar, it seems like you've used a lots of `invalid` keys around there. 
Please make sure to go through the [**LinkFree Docs**](https://linkfree.eddiehub.io/docs/quickstart) and keep the code format according to that. 

Here is a full profile example of what it should look like. https://linkfree.eddiehub.io/docs/full-profile-example

Feel free to ask if you need any help with this. :)

**Comment by SJ-Kumar** [comment]
> Hey @SJ-Kumar, it seems like you've used a lots of `invalid` keys around there. Please make sure to go through the [**LinkFree Docs**](https://linkfree.eddiehub.io/docs/quickstart) and keep the code format according to that.
> 
> Here is a full profile example of what it should look like. https://linkfree.eddiehub.io/docs/full-profile-example
> 
> Feel free to ask if you need any help with this. :)

Thanks. I will update the profile as suggested.

File: data/SJ-Kumar.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    { "icon": "FaTwitter", "url": "https://twitter.com/Jayanth_2003" },
+    { "icon": "FaGithub", "url": "https://github.com/SJ-Kumar" }
+    
+  ],
+  "testimonials": [
+    "CBID2",
+    "Akshu-on-github",
+    "ZeeMonk-pixel",
+    "krupalitrivedi"
+  ],
```
Comment (kumarsonsoff3) [inline data/SJ-Kumar.json:24]: You might want to remove this section as you don't have any testimonials yet ig. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/SJ-Kumar.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
+{
+   "basics": {
+    "name": "S.Jayanth Kumar",
+    "label": "BTech UnderGraduate at VIT Chennai",
+    "image": "https://github.com/SJ-Kumar/SJ-Kumar/blob/main/Jayanth.jpg",
```
Comment (Krishnapro) [inline data/SJ-Kumar.json]:
```diff
- "image": "https://github.com/SJ-Kumar/SJ-Kumar/blob/main/Jayanth.jpg",
+ 
```

Remote the line 5 image or Avatar will auto update with your github profile

**Review by Krishnapro — CHANGES_REQUESTED** [review]

File: data/SJ-Kumar.json (hunk 1 above)
Comment (SJ-Kumar) [reply data/SJ-Kumar.json:24]: Sure, for now, I will remove. After I gather all the required details. I will add this section.

File: data/SJ-Kumar.json (hunk 2 above)
Comment (SJ-Kumar) [reply data/SJ-Kumar.json]: I had replaced with the updated version. Please verify the latest version.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**